CODE_OF_CONDUCT-fr.md

```diff
@@ -0,0 +1,53 @@
+# Code de Conduite Contributeurs
+
+En tant que contributeurs et responsables de ce projet, et dans l'intérêt
+de favoriser une communauté ouverte et accueillante, nous nous engageons à
+respecter toutes les personnes qui contribuent en rapportant des erreurs,
+en postant des demandes de fonctionnalités nouvelles, en mettant à jour la
+documentation, en soumettant des _pull requests_ ou des correctifs, ainsi que
+toutes autres activités.
+
+Nous sommes déterminés à rendre toute participation à ce projet une
+expérience exempte de harcèlement pour tout le monde, quel que soit le niveau
+d'expérience, le sexe, l'identité ou l'expression de genre, l'orientation
+sexuelle, le handicap, l'apparence personnelle, la taille physique, la race,
+l'origine ethnique, l'âge, la religion ou la nationalité.
+
+Exemples de comportements non acceptables :
+
+* l'utilisation de langage ou d'imagerie sexualisés ;
+* les attaques personnelles ;
+* le _trolling_, ou les commentaires insultants ou désobligeants ;
+* le harcèlement en public ou en privé ;
+* la publication d'informations privées de tierces personnes, 
+  telles que les adresses physiques ou électroniques, sans permission explicite ;
+* toute conduite non professionnelle ou contraire à l'éthique.
+
+Les mainteneurs du projet ont le droit et la responsabilité de supprimer,
+modifier ou rejeter les commentaires, _commits_, code, modifications du wiki,
+questions et autres contributions qui ne respectent pas ce Code de Conduite,
+ou de bannir temporairement ou définitivement tout contributeur à la suite
+d'autres comportements qu'ils jugent inappropriés, menaçants, injurieux,
+ou nuisibles.
+
+En adoptant ce Code de Conduite, les mainteneurs du projet s'engagent à
+appliquer équitablement et uniformément ces principes à tous les aspects de
+la gestion de ce projet. Les mainteneurs de projets qui ne suivent pas ou ne
+font pas respecter le Code de Conduite peuvent être retirés de façon permanente
+de l'équipe de projet.
+
+Ce Code de Conduite s'applique à la fois au sein des espaces de projet
+ainsi que dans les espaces publics quand un individu représente le projet
+ou sa communauté.
+
+Les instances de comportement abusif, harcelant ou autrement inacceptable
+peuvent être signalés en contactant un responsable de projet à
+victorfelder at gmail.com. Toutes les plaintes seront examinées et étudiées
+et se traduiront par une réponse jugée nécessaire et appropriée aux
+circonstances. Les mainteneurs s'obligent à garder confidentielles les
+informations de la personne qui remonte un incident.
+
+Ce Code de Conduite est adaptée du [Contributor Covenant](https://.contributor-covenant.org),
+version 1.3.0, disponible à https://.contributor-covenant.org/fr/version/1/3/0/code-of-conduct.html
+
+[page d'accueil]: https://contributor-covenant.org
```

Comment:
- Fix links to be rendered :wink:
- Use right markdown syntax to link as foo notes

```suggestion
Ce Code de Conduite est adaptée du [Contributor Covenant](homepage),
version 1.3.0, disponible à https://contributor-covenant.org/fr/version/1/3/0/code-of-conduct.html

[homepage]: https://www.contributor-covenant.org
```